lib/helper.js

```diff
@@ -16,8 +18,26 @@ function isEmpty(value) {
   return typeof value == 'string' && !value.trim() || typeof value == 'undefined' || value === null;
 }
 
+function encodeKey(metricName, tags) {
+  if (isEmpty(tags)) {
+    return metricName;
+  }
+  // Sort the tags based on their keys to ensure we don't duplicate metrics if the same tags come out of order.
+  var tagsJson = JSON.stringify(tags, Object.keys(tags).sort());
```

Comment:
I don't know what your intention is here, but I don't think this does what you think it does.  The second param for stringify is a whitelist for which keys to include.

Reply:
so basically it just sorts the tags kv pairs based on their keys.